### PR TITLE
fix(TSEO-32): Added page number in meta data

### PR DESCRIPTION
**Pull request by virendrapatelnuk**
For any search query in topic, page numbers are changing but the title and description remain the same. To differentiate and let the search engines know that the page has changed, we suggest adding page numbers at the end in the title and description fields.

Currently if topic is India URL https://www.thetimes.co.uk/topic/india Title: India | The Times & The Sunday Times Description: Discover expert India articles from The Times and The Sunday Times. 

*Suggested: * Title: India | The Times & The Sunday Times | Page Number

Description: Discover expert India articles from The Times and The Sunday Times. Page Number of total pages  

[JIRA LINK](https://nidigitalsolutions.jira.com/browse/TDP-172)

### CODE REVIEW

**Comment by bibicollins1** [comment]
Hi @virendrapatelnuk Just wondering about the sentence structure on the description, maybe it's more a case of getting key words if so it's fine, but if not I would think 'Discover expert articles about India from The Times and The Sunday Times. Page Number of total pages' might work a bit better as it would work for topics like Politics but also place names and people names etc? I might just be being pedantic though.

File: packages/topic/__tests__/web/__snapshots__/topic-loading.web.test.js.snap [hunk 1]
```diff
@@ -4,11 +4,10 @@ exports[`1. an article list loading 1`] = `
 <topicArticlesProvider>
   <Helmet>
     <title>
-      
        | The Times & The Sunday Times
     </title>
     <meta
-      content="Discover expert  articles from The Times and The Sunday Times."
+      content="Discover expert  articles from The Times and The Sunday Times. Page 1 of 0"
```
Comment (bibicollins1) [inline packages/topic/__tests__/web/__snapshots__/topic-loading.web.test.js.snap:10]: Should this be page 1 of 1?

File: packages/topic/__tests__/web/__snapshots__/topic.web.test.js.snap [hunk 2]
```diff
@@ -48,15 +47,14 @@ exports[`3. an article list 1`] = `
 <topicArticlesProvider>
   <Helmet>
     <title>
-      Test Name
-       | The Times & The Sunday Times
+      Test Name | The Times & The Sunday Times | Page 2
     </title>
     <meta
-      content="Test italic bold text."
+      content="Test italic bold text. Page 2 of 1"
```
Comment (bibicollins1) [inline packages/topic/__tests__/web/__snapshots__/topic.web.test.js.snap:53]: Same here, there are some others on lines 138 and 228

File: packages/topic/__tests__/web/__snapshots__/topic.web.test.js.snap [hunk 3]
```diff
@@ -6,11 +6,10 @@ exports[`2. a loading state 1`] = `
 <topicArticlesProvider>
   <Helmet>
     <title>
-      
        | The Times & The Sunday Times
     </title>
     <meta
-      content="Discover expert  articles from The Times and The Sunday Times."
+      content="Discover expert  articles from The Times and The Sunday Times. Page 1 of 1"
```
Comment (bibicollins1) [inline packages/topic/__tests__/web/__snapshots__/topic.web.test.js.snap:12]: If there is no topic do we want to remove the empty string space here?

File: packages/topic/__tests__/web/__snapshots__/topic-loading.web.test.js.snap (hunk 1 above)
Comment (virendrapatelnuk) [reply packages/topic/__tests__/web/__snapshots__/topic-loading.web.test.js.snap:10]: @bibicollins1 Yes! it's my bad. I'll change it.

Comment (virendrapatelnuk) [reply packages/topic/__tests__/web/__snapshots__/topic-loading.web.test.js.snap]: @bibicollins1 
Done, kindly review.

**Pull request merged**